README.md

```diff
@@ -1,2 +1,59 @@
 # HelloMicroverse
-First repository at Microverse
+
+- First repository setup at Microverse. Added the html/css linter to the repository. It includes a basic html file with a h1 element that types the text "Hello MMicroverse!" on the browser window
+- There is also a css file that changes the text-color to lightblue.
+
+## Built with
+
+- HTML
+- CSS
+
+## Setup for Basic HTML/CSS Project
+
+- Linter for HTML/CSS installed
+
+## Getting Started
+
+### Prerequisites
+
+- Text editor
+- Basic html/css knowledge
+
+### Setup
+
+### Install
+
+### Usage
+
+### Deployment
+
+- [https://github.com/microverseinc/linters-config/tree/master/html-css](Linter)
+
+## Authors
+
+👤 **Author1**
+
+- Github: [@itsmraga-hub](https://github.com/itsmraga-hub)
+
+- Twitter: [@RagaMacharia](https://twitter.com/RagaMacharia)
+
+- Linkedin: [@itsmraga](https://www.linkedin.com/in/itsmraga/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse team
+- Inspiration
+
+## License
+
+None.
```

Comment:
- [ OPTIONAL] I can see here that you didn't add the MIT.md license so you can put it here [MIT.md](https://github.com/lorenzoworx/Portfolio/blob/main/MIT.md) this is an example you can update it and put your name.